perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ResourcesManagerMethod.java

```diff
@@ -294,15 +293,19 @@ public Vo call(ApiCaller ac, Deserializer parms) throws PerunException {
 		public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 			parms.stateChangingCheck();
 
+			boolean async = parms.contains("async") ? parms.readBoolean("async") : false;
```

Comment:
I think the new parameter should be also added to these methods in `openapi` so that we can call it with the parameter from GUI.